vllm/model_executor/model_loader/weight_utils.py

```diff
@@ -252,6 +251,7 @@ def download_weights_from_hf(
 def download_safetensors_index_file_from_hf(
     model_name_or_path: str,
     cache_dir: Optional[str],
+    index_file: str,
```

Comment:
super nit: I think it makes more sense to have cache_dir after index_file, similar to how hf_hub_download is called

Reply:
happy to change